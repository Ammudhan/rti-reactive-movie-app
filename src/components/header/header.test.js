import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Header from './header';
import {BrowserRouter as Router} from "react-router-dom";

describe('Header', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <Router>
                <Header />
            </Router>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('matches snapshot', () => {
        const
            tree = renderer.create(
                <Router>
                    <Header />
                </Router>
            ).toJSON();
        expect(tree).toMatchSnapshot();
    });

});