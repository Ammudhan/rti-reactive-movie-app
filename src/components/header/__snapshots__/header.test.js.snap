// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should matches snapshot 1`] = `
<span
  className="row"
>
  <span
    className="col-12 page-layout d-flex align-items-center justify-content-between"
  >
    <h1
      className="title"
    >
      Pop Movie
    </h1>
    <span
      className="nav-icon"
    />
  </span>
</span>
`;

exports[`Header matches snapshot 1`] = `
<span
  className="row"
>
  <span
    className="col-12 page-layout d-flex align-items-center justify-content-between"
  >
    <h1
      className="title"
    >
      Pop Movie
    </h1>
    <span
      className="nav-icon"
    />
  </span>
</span>
`;
